includes/class-sensei-lesson.php

```diff
@@ -2577,16 +2577,27 @@ public function lesson_remove_multiple_questions() {
 		$question_id_to_remove      = $question_data['question_id'];
 		$quiz_id_to_be_removed_from = $question_data['quiz_id'];
 
-		// remove the question from the lesson quiz
-		$quizzes = get_post_meta( $question_id_to_remove, '_quiz_id', false );
-		foreach ( $quizzes as $quiz_id ) {
-			if ( $quiz_id == $quiz_id_to_be_removed_from ) {
+		if ( 'multiple_question' !== get_post_type( $question_id_to_remove ) ) {
+			die( '' );
+		}
+
+		$found_quiz = false;
+		$quizzes    = get_post_meta( $question_id_to_remove, '_quiz_id', false );
```

Comment:
Just a note: currently there doesn't seem to be a way for these `mulitple_question` posts to belong to multiple quizzes.